Ollivanders/src/net/pottercraft/Ollivanders2/Book/VOYAGES_WITH_VAMPIRES.java

```diff
@@ -16,8 +16,8 @@ public VOYAGES_WITH_VAMPIRES ()
       title = shortTitle = "Voyages with Vampires";
       author = "Gilderoy Lockhart";
       branch = O2MagicBranch.DARK_ARTS;
-
-      spellList.add(Spells.DEPULSO);
-      spellList.add(Spells.POINT_ME);
+      
+      spells.add(Spells.DEPULSO);
+      spells.add(Spells.OBLIVIATE);
```

Comment:
You didn't put POINT_ME anywhere

Reply:
Added to Quintessence - 5th year Charms book.